### PR TITLE
Ta 31

### PR DESCRIPTION
**Pull request by liran26**

### CODE REVIEW

File: components/ProductCard.js [hunk 1]
```diff
@@ -135,11 +135,13 @@ const ProductCard = ({ productId }) => {
...
       {/* Add to Cart button */}
       {product && (
-        <Pressable style={styles.addToCart}>
-          <Text style={{ color: "white", textAlign: "center" }}>
-            ADD TO CART
-          </Text>
-        </Pressable>
+        <View style={styles.addToCart}>
+          <Pressable style={styles.addToCartButton}>
+            <Text style={{ color: "white", textAlign: "center" }}>
```
Comment (GiliEllen) [inline components/ProductCard.js]: add style

File: screens/productScreen/ChooseColorModal/ChooseColorModal.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      <Pressable
+        style={[styles.choosing, colorPressed && styles.choosingPressed]}
+        onPress={() => {
+          setColorPressed(!colorPressed);
+          setModalVisibility(true);
+        }}
+      >
+        {selectedColor !== "Color" ? (
+          <View
+            style={{ width: 20, height: 20, backgroundColor: selectedColor }}
```
Comment (GiliEllen) [inline screens/productScreen/ChooseColorModal/ChooseColorModal.js]: add style

File: screens/productScreen/ProductScreen.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ import ProductCard from '../../components/ProductCard';
 const ProductScreen = ({ navigation }) => {
     return (
         <View style={styles.container}>
-            <ProductCard navigation={navigation} />
+            <ProductCard navigation={navigation} productId={"65faf26b8dec99b4cd38a768"}/>
```
Comment (GiliEllen) [inline screens/productScreen/ProductScreen.js]: remove

**Review by GiliEllen — CHANGES_REQUESTED** [review]

File: screens/productScreen/ChooseColorModal/ChooseColorModal.js (hunk 2 above)
Comment (liran26) [reply screens/productScreen/ChooseColorModal/ChooseColorModal.js]: i couldn't find way to pass the selectedColor to style if im not using inline style. i tried to sepperate widh and height from background color and use inline style only for the background and it didn't work

Comment (liran26) [reply screens/productScreen/ChooseColorModal/ChooseColorModal.js]: never mind, i fixed it :)

**Review by GiliEllen — APPROVED** [review]

**Pull request merged**